Specs-state (#290)
PrestaShop/prestashop-specs: content/1.7/back-office/international/locations/add-new-state.md

> ---

Reviewer: Could you rename this file "add-new-state.md" by "add-edit-new-state.md" ?

Reviewer: Also the title and titles

Reviewer: @Saimis777 I reopened this comment because it's not solved

Reviewer: Also, you should create a "states" folder: content/1.7/back-office/international/locations/states/add-edit-state.md

> title: Add new state

Reviewer: ```suggestion
title: Add edit state
```

weight: 1
---

> # Add new state

Reviewer: ```suggestion
# Add edit state
```

> While adding or editing a state, 5 fields below are shown:

Reviewer: ```suggestion
While adding or editing a state, the following fields/parameters are displayed:
```

>  - **Name** - this is a mandatory field. A tooltip for this field shows: "Provide the state name to be displayed in addresses and on invoices.". When trying to save with the field empty will show an error message: "The name field is required.".

Reviewer: ```suggestion
 - **Name** - this is a mandatory field. A tooltip for this field is displayed: "Provide the state name to be displayed in addresses and on invoices.". When trying to save and if the field is empty, the following error message is displayed: "The name field is required.".
```

Reviewer: what happens if I try to save without filling this field?

Author: Is there something more to be added? As I've added already the error message when field is empty

>  - **ISO code** - this is a mandatory field. A tooltip for this field shows: "1 to 4 letter ISO code. You can prefix it with the country ISO code if needed.". When trying to save with the field while it's empty, will show an error message:  "The iso_code field is required."

Reviewer: ```suggestion
 - **ISO code** - this is a mandatory field. A tooltip for this field is displayed: "1 to 4 letter ISO code. You can prefix it with the country ISO code if needed.". When trying to save and if the field is empty, the following error message is displayed:  "The iso_code field is required."
```

Reviewer: what happens if I try to save without filling this field?

>  - **Country** - this is a mandatory drop-down field. A tooltip for this field shows: "Country where the state is located. Only the countries with the option "contains state" enabled are displayed."

Reviewer: ```suggestion
 - **Country** - this drop-down is mandatory. A tooltip for is displayed: "Country where the state is located. Only the countries with the option "contains state" enabled are displayed."
```

Reviewer: What is the country selected by default ?

>  - **Zone** - this is a mandatory drop-down field. A tooltip for this field shows: "Geographical region where this state is located. Used for shipping".

Reviewer: ```suggestion
 - **Zone** - this drop-down is mandatory. A tooltip is displayed: "Geographical region where this state is located. Used for shipping".
```

Reviewer: What is the zone selected by default ?

>  - **Status** - this is a mandatory toggle switch. It can be turned to "yes" or "no".

Reviewer: ```suggestion
 - **Status** - this toggle is mandatory. It can be turned to "yes" or "no".
```

Reviewer: What is the default state for this toggle ?

## Call to action

 There are 2 buttons at the bottom of this page: 

>  - Cancel: The state  is not added/edited and the listing is displayed, with a message: "Successful creation.".

Reviewer: It's the cancel button, you can't have a message "Successful creation.", otherwise it's a bug

>  - Save: In the case of adding a state, the state is added and a success message is displayed on the listing: "Successful creation.". In the case of editing a state, the attribute is edited and a success message is displayed on the listing: "Successful update.".

Reviewer: Bad copy/past here it's not about attribuite
